var/spack/repos/builtin/packages/py-gpaw/package.py

```diff
@@ -26,11 +29,15 @@ class PyGpaw(PythonPackage):
     depends_on('mpi', when='+mpi', type=('build', 'link', 'run'))
     depends_on('python@2.6:', type=('build', 'run'), when='@:1.3.0')
     depends_on('python@3.5:', type=('build', 'run'), when='@19.8.1:')
+    depends_on('python@3.6:', type=('build', 'run'), when='@20.10.0:')
     depends_on('py-ase@3.13.0:', type=('build', 'run'), when='@1.3.0')
     depends_on('py-ase@3.18.0:', type=('build', 'run'), when='@19.8.1')
+    depends_on('py-ase@3.19.0:', type=('build', 'run'), when='@20.1.0')
+    depends_on('py-ase@3.20.1:', type=('build', 'run'), when='@20.10.0')
+    depends_on('py-ase@3.21.0:', type=('build', 'run'), when='@21.1.0')
```

Comment:
gpaw 21.1.0 requires ase 3.20.1:

Reply:
No. ase 3.21.0 is required per the official release notes https://wiki.fysik.dtu.dk/gpaw/releasenotes.html#version-21-1-0

Reply:
Hmm, I tend to trust the `setup.py` more than the documentation, but I suppose they might have forgotten to update it...